addons/web/static/src/public/colibri.js

```diff
@@ -174,13 +178,16 @@ export class Colibri {
         }
     }
 
-    applyAttr(el, attr, value) {
+    applyAttr(el, attr, value, initialValue) {
         if (attr === "class") {
             if (typeof value !== "object") {
                 throw new Error("t-att-class directive expects an object");
             }
             for (const cl in value) {
-                for (const c of cl.trim().split(" ")) {
+                for (let c of cl.trim().split(" ")) {
+                    if (value[cl] === this.INITIAL_VALUE) {
+                        value[cl] = initialValue[cl];
```

Comment:
That might pollute something used outside. Maybe extract `let singleValue = value[cl];` then replace that one.